DuckDuckGo/AppConfigurationFetch.swift

```diff
@@ -68,9 +70,6 @@ class AppConfigurationFetch {
     @UserDefaultsWrapper(key: .lastConfigurationRefreshDate, defaultValue: .distantPast)
     static private var lastConfigurationRefreshDate: Date
 
-    @UserDefaultsWrapper(key: .backgroundFetchTaskExpirationCount, defaultValue: 0)
-    static private var backgroundFetchTaskExpirationCount: Int
```

Comment:
Shouldn't we also remove `backgroundFetchTaskExpirationCount`  Key? This seems to be defined in two places now - as an enum in `UserDefaultsWrapper.Key` and as a static let in `AppUserDefaults.Keys`? Also maybe we should be reusing the old string key as sort of migration?